pkgs/development/libraries/teensy-cmake-macros/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config }:
+
+stdenv.mkDerivation {
+  pname = "teensy-cmake-macros";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "newdigate";
+    repo = "teensy-cmake-macros";
+    rev = "dc401ed23e6e13a9db3cd2a65f611a4738df3b0e";
+    hash = "sha256-E+BOlsCJtOScr3B5GSv1WM6rFv6cFYvm/iJ893fsmXM=";
+  };
+
+  propagatedBuildInputs = [ cmake pkg-config ];
+
+  setupHook = ./setup-hook.sh;
```

Comment:
I suggest you to put this in a passthru.
Look at our code packaging Zig 0.11 as an example.

Reply:
Thanks, I'll take a look.